things/service.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/mainflux/mainflux/pkg/errors"
 
 	"github.com/mainflux/mainflux"
-	uuidProvider "github.com/mainflux/mainflux/pkg/uuid"
+	mfxulid "github.com/mainflux/mainflux/pkg/ulid"
```

Comment:
Why `mfxulid` and not `ulidProvider` or `IDProvider`? Maybe we can also remove everything and use just `ulid`?